tests/simpleConnectionTest.py

```diff
@@ -0,0 +1,9 @@
+from HCPInterface import WD
+from HCPInterface.hcp import HCPManager
```

Comment:
So is it HCPinterface or NGPinterface? Correct name going forward?

Reply:
Will do it in a sep PR since itll be a bit of a pain